damus/ContentView.swift

```diff
@@ -282,7 +282,10 @@ struct ContentView: View {
                                 
                                 ToolbarItem(placement: .navigationBarTrailing) {
                                     HStack(alignment: .center) {
-                                        SignalView(state: damus_state!, signal: home.signal)
+                                        if let pk = self.active_profile {
```

Comment:
You don't need the `self.` here, and you can use the new abbreviated syntax:
```swift
if let active_profile {
    ...
}
```

Reply:
Amended.

Reply:
Put this back in as I thought it was causing a build error. May not have been actually.

Reply:
No build error. And you don't event need a separate named entity if you do it like this:
```swift
if let active_profile {
    let profile_model = ProfileModel(pubkey: active_profile, damus: damus_state!)
    SignalView(state: damus_state!, signal: home.signal, profile: profile_model)
}
```